### ISSUE TITLE
Slider images aren't secure

### ISSUE THREAD

**Issue by twistedindustries** [Magestore/Bannerslider-Magento2#64]
My bannerslider wasn't returning secure urls when it is supposed to.  I resolved this by adding the following in Magestore\Bannerslider\Helper\Data.php
`/**
     * Check if current requested URL is secure
     *
     * @return boolean
     */    
    public function isCurrentlySecure()
    {
        return $this->_storeManager->getStore()->isCurrentlySecure();
    }`

then in Magestore\Bannerslider\Block\Item.php in the function getBannerImageUrl I changed the return to:

`return $this->_bannersliderHelper->getBaseUrlMedia($banner->getImage(), $this->_bannersliderHelper->isCurrentlySecure());`

**Comment by surfer190** [comment]
@twistedindustries Have you created a pull request for the fix?

**Comment by twistedindustries** [comment]
@surfer190 Nope, I just fixed it locally.  No pull request.  Your more than welcome to create one using my code, I figured this would just be fixed in later releases.

**Comment by developer-lindner** [comment]
@twistedindustries good job!

**Comment by surfer190** [comment]
@developer-lindner I don't know if you think you are contributing but you aren't. @twistedindustries I would suggesting opening a pull request and fixing if you are so inclined. I would appreciate.

**Comment by twistedindustries** [comment]
@surfer190 The only thing that has stopped me is being new with git.  Recently I've been using it a little more so I'll attempt that.

**Comment by sjanjic** [comment]
There is pull request for this fix, is it going to be merged soon?
